stream/stream_impl.h

```diff
@@ -142,6 +142,20 @@ class StreamPublisherImpl {
     return result;
   }
 
+  template <current::locks::MutexLockStatus MLS>
+  idxts_t PublisherPublishUncheckedImpl(std::string&& entry_json) {
+	  const auto result = data_->persister.template PersisterPublishUncheckedImpl<MLS>(std::move(entry_json));
```

Comment:
`PersisterPublishUncheckedImpl` has only one version accepting `const std::string&`. Why do you need `std::string&&` versions of Publish implementations?